content/posts/packit-030.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Packit 0.3.0"
+date: 2019-04-10
+draft: true
+---
+
+In [previous post](https://packit.dev/posts/packit-020/) we promised to release a new release every 2 weeks and we are already breaking that as it's been 3 weeks since then.
```

Comment:
```suggestion
In the [previous post](https://packit.dev/posts/packit-020/) we promised to provide a new release every 2 weeks and we are already breaking that as it's been 3 weeks since then. The reason is that we wanted to add the full source-git support into 0.3.0.
```

Reply:
Sadly, I forgot to add the `--upstream-revision` option to CLI, so it's only in the PackitAPI. That means we do not have full source-git support in 0.3.0. Sorry about that.